cli/sawtooth_cli/cluster.py

```diff
@@ -301,6 +300,10 @@ def do_cluster_start(args):
 
     save_state(state)
 
+    # Make sure the docker cluster network is up before starting nodes
+    if isinstance(node_controller, DockerNodeController):
```

Comment:
What about changing the test condition to something like `if manage_type == 'docker':`?